Zend/tests/gh13145.phpt

```diff
@@ -0,0 +1,16 @@
+--TEST--
+GH-13145: strtok() misoptimization
+--FILE--
+<?php
+$tok = strtok("This is\tan example\nstring", " \n\t");
```

Comment:
shouldn't the `strtok` function impl. be fixed instead?

Reply:
`strtok()` is stateful, the call cannot be skipped.